### ISSUE TITLE
Make handler optional?

### ISSUE THREAD

**Issue by DeRain** [GeniusesOfSymfony/PubSubRouterBundle#12]
Hi guys, the package is amazing! I'm currently using it for route MQTT topics and it's helped to save a lot of time.

In my case i don't need to define callback at all - I just need the ability to generate routes from defined YML rules.

Don't you think about making the handler param optional? For now, i should define an empty handler in every route. 

For example:
```yaml
user_order_notification:
    channel: order/status/{order_id}/user/{user_id}
    handler: #required for now
    requirements:
        order_id:
          pattern: "d+"
        user_id:
          pattern: "d+"
```

Thanks!

**Comment by jjsaunier** [comment]
Yes, it make sense for that use case, indeed. 

If you can submit a PR, you have to modify around here : 

https://github.com/GeniusesOfSymfony/PubSubRouterBundle/blob/master/Loader/YamlFileLoader.php#L51

to make the key handler optional :)

**Comment by DeRain** [comment]
@ProPheT777 I will make a PR ASAP. Thank you for the hint :)

**Comment by DeRain** [comment]
Hi @ProPheT777. Just made a pull request.
#13

**Comment by jjsaunier** [comment]
released as v0.3.0, thank you @DeRain
